### ISSUE TITLE
Put the raw files in instead of a zip file

### ISSUE THREAD

**Issue by AnachronicNomad** [Studio528-real/Broken-Tools#1]
It would be neat to view the files raw, and may make it easier to use version control like this as you edit/add each component.

You can also specify ["Release Milestones"](https://help.github.com/en/articles/creating-releases) (check out the "Further reading" section at the bottom as well, which can auto-magically package a zip file for you.

**Comment by Studio528-real** [comment]
the way minecraft uses resource/texture packs is as a .zip file not an individual folder for files, unless you mean a way to view the textures themselves without downloading the zip,

**Comment by AnachronicNomad** [comment]
Yes. That is correct. Git will track different versions of the .json files, in addition to the actual .pngs themselves as you change them over time. 

Github, through its "releases" mechanism, allows for you to upload a .zip that is usable by minecraft, in addition to taking the entire directory and packaging it as a .zip for you.
